mora/service/address.py

```diff
@@ -333,12 +333,14 @@ def convert(start, end, effect):
                 if not c.is_effect_relevant(addrrel['virkning']):
                     continue
 
-                yield {
-                    **get_one_address(c, addrrel, class_cache),
-                    keys.ADDRESS_TYPE: get_address_class(c, addrrel,
-                                                         class_cache),
-                    keys.VALIDITY: common.get_effect_validity(addrrel),
-                }
+                try:
+                    addr = get_one_address(c, addrrel, class_cache)
+                except KeyError as e:
```

Comment:
Where is KeyError thrown? Are we sure we're not swallowing actual exceptions here?

Reply:
> Where is KeyError thrown? Are we sure we're not swallowing actual exceptions here?

In [`get_one_class()`](https://github.com/magenta-aps/mora/pull/87/commits/6d113c1ac5bbf45ba28ef130fd13fcb3c2cf9874#diff-8426190c255ed9b2d05701905cef41c1R132) — we _could_ also return `None` there and handle that, but an exception seemed more appropriate for what ought to be an exceptional case…

Reply:
As far as I can tell from this, the UI is unaware that something went wrong. Should they be aware? Or do we just expect whoever is in charge to check the log ever so often for errors?